### PR TITLE
attempted fix

### PR DESCRIPTION
**Pull request by zorro1rr**
I have never coded in shell script, and I just saw that someone else already sent a pull request but I guess it can't hurt to submit a request! Feel free to contact me on slack, thanks!

### CODE REVIEW

File: _webi/template.sh [hunk 1]
```diff
@@ -168,6 +169,8 @@ webi_download() {
     fi
 
     mv "$my_dl.part" "$my_dl"
+
```
Comment (solderjs) [inline _webi/template.sh:172]: just add an `echo ""` here as well

**Comment by solderjs** [comment]
FYI it's good practice to tag the issue in the description of the PR/MR. i.e. #63

**Comment by solderjs** [comment]
If you'll add that echo then I'll approve and merge your change, and then tomorrow (or whenever), I can merge in the other one as well with the formatting update.

**Comment by solderjs** [comment]
Ended up I merged both. I didn't catch this earlier but yours had the correct "$my_url" variable whereas the other did not.

**Comment by zorro1rr** [comment]
Cool, sorry about the delay on that, I just saw the email notification.

**Comment by solderjs** [comment]
Forgot to put this in prod the other day. It's live now.

**Pull request merged**